python/cudf/cudf/pandas/__init__.py

```diff
@@ -25,14 +25,16 @@ def install():
     global LOADED
     LOADED = loader is not None
 
-    if (rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", None)) is not None:
+    rmm_mode = os.getenv("CUDF_PANDAS_RMM_MODE", "managed_pool")
+    if rmm_mode != "disable":
         # Check if a non-default memory resource is set
         current_mr = rmm.mr.get_current_device_resource()
         if not isinstance(current_mr, rmm.mr.CudaMemoryResource):
             warnings.warn(
                 f"cudf.pandas detected an already configured memory resource, ignoring 'CUDF_PANDAS_RMM_MODE'={str(rmm_mode)}",
```

Comment:
this warnings doesn't seem accurate. We need some sort of early return in place in order to ignore the env var, otherwise we will process it regardless of what the current mr is.

Reply:
Introduced an early return.